Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -195,64 +195,77 @@ internal async ValueTask<List<Result>> RequestUpdateAsync(string search, Cancell
                     }
                 };
 
-            var results = resultsForUpdate
-                .Select(x =>
-                    new Result
-                    {
-                        Title = $"{x.Name} by {x.Author}",
-                        SubTitle = $"Update from version {x.CurrentVersion} to {x.NewVersion}",
-                        IcoPath = x.IcoPath,
-                        Action = e =>
-                        {
-                            string message = string.Format(
-                                Context.API.GetTranslation("plugin_pluginsmanager_update_prompt"),
-                                x.Name, x.Author,
-                                Environment.NewLine, Environment.NewLine);
+            var updateAllResult = new Result
+            {
+                Title = "Update all plugins",
+                SubTitle = "Update all plugins to the latest version",
+                IcoPath = icoPath,
+                Action = e => 
+                {
+                    // TODO: logic here
+                    return true;
+                },
+                ContextData = new UserPlugin()
+            };
 
-                            if (MessageBox.Show(message,
-                                    Context.API.GetTranslation("plugin_pluginsmanager_update_title"),
-                                    MessageBoxButton.YesNo) == MessageBoxResult.Yes)
+            var results = new[] {updateAllResult}.Concat(
```

Comment:
Maybe someone has an idea how to make it more elegant. It works to set the top result this way, but I guess it's not the most elegant way